kubectl-plugin/README.md

```diff
@@ -2,56 +2,23 @@
 
 Kubectl plugin/extension for Kuberay CLI that provides the ability to manage ray resources.
 
-## Installation
-
-<!-- 1. Check [release page](https://github.com/ray-project/kuberay/releases) and download the necessary binaries. -->
-1. Run `go build cmd/kubectl-ray.go`
-2. Move the binary, which will be named `kubectl-ray` to your `PATH`
-
 ## Prerequisites
 
 1. Make sure there is a Kubernetes cluster running with KubeRay installed.
 2. Make sure `kubectl` has the right context.
 
-## Usage
-
-### Retrieve Ray Clusters
-
-    Usage:
-    ray cluster get [flags]
-
-    Aliases:
-    get, list
-
-    Description:
-    Retrieves the ray cluster information.
-
-    ```
-    $ kubectl ray cluster get
-    NAME                       NAMESPACE   DESIRED WORKERS   AVAILABLE WORKERS   CPUS   GPUS   TPUS   MEMORY   AGE
-    random-kuberay-cluster   default     1                 1                   5      1      0      24Gi     13d
-    ```
+## Installation
 
-### Retrieve Ray Cluster Head Logs
+1. Install [Krew](https://krew.sigs.k8s.io/docs/user-guide/setup/install/).
+2. (TODO: Replace this step with the installation command).
```

Comment:
Do we plan to add this instruction before merging this PR?

Reply:
I am considering distributing our kubectl plugin with `krew`, but this requires we to
1. Package our plugin and then put it on GitHub release page
2. Open a pull request to krew index github repo

So I think it's better to merge this PR so that we can implement dynamic completion of nouns.

See https://krew.sigs.k8s.io/docs/developer-guide/distributing-with-krew/ for details.

Reply:
I like the idea of using Krew, but until it's published we should still providing working solution here to install the plugin even if that involves manually compiling it. What do you think?

Reply:
There is an offline slack discussion here:

https://ray.slack.com/archives/C07M825T94Z/p1726476910442009

The message is:

I just found [this repository](https://github.com/rajatjindal/krew-release-bot?tab=readme-ov-file), which can automatically release our plugin to Krew. However, it relies on git release tags. Currently KubeRay is a monorepo, with all Kubernetes-related items stored in it. So, I'm thinking that the tag format might need to be something like kubectl-ray/v0.1.0, and the versioning might need to be separate from the ray-operator version. I'm just not sure if this will look odd because I noticed that the current tags match the release tags, such as ray-operator/v1.2.1 and v1.2.1. Does anyone have any thoughts on this?
https://github.com/ray-project/kuberay/tags

Reply:
I moved the manually installation guide to `Development` section.

Reply:
Also before the PR to Krew index repo is merged, we can tell the users to install from our manifest file like this:

`kubectl krew install --manifest-url <url>`

Reply:
Make senses to me. Updated.